src/analyze/analyze.c

```diff
@@ -102,6 +102,7 @@ RuntimeScope arg_runtime_scope = RUNTIME_SCOPE_SYSTEM;
 RecursiveErrors arg_recursive_errors = _RECURSIVE_ERRORS_INVALID;
 bool arg_man = true;
 bool arg_generators = false;
+const char *arg_instance = "test_instance";
```

Comment:
btw, for new code i'd always use strdup() memory for such arg_xyz strings, rather than point directly into argv[]. That's because we nowadays change argv[] in rename_process() at so many places, for example in safe_fork(), which unexpectedly then changes these strings. Hence, given we only duplicate a very small number of strings when parsing command lines, i#d always strdup() this, just to avoid confusion later on (because it is *very* confusing, if various arg_* strings become invalid after safe_fork())

Reply:
OK. Maybe better to update CODING_STYLE?